src/js/utils/Event.js

```diff
@@ -7,6 +7,7 @@
     if (typeof el === 'string') {
       el = document.querySelector(el);
     }
+    scope = scope || this;
```

Comment:
We can throw a more explicit error, but I'd prefer to avoid this silent behaviour change.